### ISSUE TITLE
pickling subclassed XMLSchema instances fails

### ISSUE THREAD

**Issue by leo-b** [sissaschool/xmlschema#263]
subclassed XMLSchema instances that are defined in separate modules currently cannot be pickled because the generated `Meta...` class is stored in the wrong module (in :

```python
>>> import xmlschema
>>> import pickle
>>> class MyXMLSchema(xmlschema.XMLSchema):
...     pass
... 
>>> s = MyXMLSchema('git/xmlschema/xmlschema/schemas/XML/xml_minimal.xsd')
>>> p = pickle.dumps(s)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
_pickle.PicklingError: Can't pickle <class '__main__.MetaMyXMLSchema'>: attribute lookup MetaMyXMLSchema on __main__ failed
>>> xmlschema.validators.schema.MetaMyXMLSchema 
<class '__main__.MetaMyXMLSchema'>
```

**Comment by leo-b** [comment]
Fixes #263

**Comment by brunato** [comment]
Hi,
the next minor release the XMLSchema's metaclass has been partially rewritten, removing BUILDERS dict (deprecated), so I've added also some pickling tests for schema subclasses.

**Comment by brunato** [comment]
Fixed with v1.8.0
